cap-notebook/demoapp/_i18n/i18n_de.properties

```diff
@@ -0,0 +1,15 @@
+Books = Bücher
```

Comment:
These translations are provided by Translation team right ?

Reply:
These were pushed by mistake. I have removed them. Thanks for pointing out.